plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol

```diff
@@ -119,6 +121,8 @@ contract PaymentInFlightExitRouter is IExitProcessor, Operated, OnlyWithValue {
     )
         public
     {
+        maintainer = framework.getMaintainer();
```

Comment:
Can we consider using `framework.getMaintainer()` in `onlyFrom()` instead of creating another storage variable?